npmVowelCounter/README.md

```diff
@@ -0,0 +1,11 @@
+# Vowel Counter install
+
+This will allow you to easily count the amount of vowels in a string.
+
+Install instructions:
+
+open terminal 
+npm install vowelcounter
+add const "countVowels = require('vowelcounter');" to the top of your index.js file.
+test example countVowels("Hello World);
```

Comment:
You could improve the formatting of the docs using ``` marks, and denoting a language:
```js
const countVowels = require('vowelcounter');
const vowelTotal = countVowels('Hello World');
```